modules/hm/desktops/random-theme.sh

```
@@ -0,0 +1,77 @@
+  -r, --random-theme    Apply a random theme
+  -a, --all             Apply a random theme and a random wallpaper
+
+Examples:
+    $0 -r               # Apply a random theme
+    $0 -a               # Apply a random theme and a random wallpaper
+EOF
+}
+
+APPLY_RANDOM_THEME=false
+APPLY_RANDOM_WALLPAPER=false
+SPECIFIED_THEME=""
```

Shellcheck (0.10.0)

[warning] SPECIFIED_THEME appears unused. Verify use (or export if used externally).

(SC2034)